kolibri/plugins/learn/assets/src/views/content-cards/content-carousel.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+
+  <section class="content-carousel">
+
+    <header v-if="header" class="content-carousel-header">
+      <h2 v-if="header"> {{header}} </h2>
+      <sub v-if="subheader"> {{subheader}} </sub>
```

Comment:
I disagree. It's just describing the content set, an `h3` implies that it's a subsection. I can look into some alternative HTML elements though